### PR TITLE
fix: panic on down/up.

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

**Review by davemooreuws — APPROVED** [review]

File: pkg/provider/pulumi/aws/aws.go [hunk 1]
```diff
@@ -98,12 +98,12 @@ func New(p *project.Project, name string, envMap map[string]string) (common.Pulu
 
 	// Hydrate from file if already exists
 	b, err := os.ReadFile(filepath.Join(p.Dir, "nitric-"+name+".yaml"))
-	if !errors.Is(err, os.ErrNotExist) {
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
```
Comment (asalkeld) [inline pkg/provider/pulumi/aws/aws.go]: good catch, but probably less messy to test errors.Is()

File: pkg/provider/pulumi/aws/aws.go [hunk 2]
```diff
@@ -98,12 +98,12 @@ func New(p *project.Project, name string, envMap map[string]string) (common.Pulu
...
 		return nil, err
 	} else if err == nil {
-		err = yaml.Unmarshal(b, asc)
-		if err != nil {
-			return nil, err
+		marshErr := yaml.Unmarshal(b, asc)
```
Comment (asalkeld) [inline pkg/provider/pulumi/aws/aws.go]: i doubt this was causing an issue

File: pkg/provider/pulumi/aws/aws.go (hunk 1 above)
Comment (tjholm) [reply pkg/provider/pulumi/aws/aws.go]: fixed

File: pkg/provider/pulumi/aws/aws.go (hunk 2 above)
Comment (tjholm) [reply pkg/provider/pulumi/aws/aws.go]: fixed.

**Review by asalkeld — APPROVED** [review]
nice

**Pull request merged**